app/lua/lflash.c

```diff
@@ -0,0 +1,224 @@
+/*
+** $Id: lflash.c
+** See Copyright Notice in lua.h
+*/
+
+#define lflash_c
+#define LUA_CORE
+#define LUAC_CROSS_FILE
+#include "lua.h"
+
+#ifdef LUA_FLASH_STORE
+#include "lobject.h"
+#include "lauxlib.h"
+#include "lstate.h"
+#include "lfunc.h"
+#include "lflash.h"
+#include "platform.h"
+#include "vfs.h"
+
+#include "c_fcntl.h"
+#include "c_stdio.h"
+#include "c_stdlib.h"
+#include "c_string.h"
+
+/*
+ * Flash memory is a fixed memory addressable block that is serially allocated by the
+ * luac build process and the out image can be downloaded into SPIFSS and loaded into
+ * flash with a node.flash.load() command. See luac_cross/lflashimg.c for the build
```

Comment:
Nitpick, the command is currently `node.flash.reload()`, and subject to iteration :)

Reply:
Yup, but as we discussed these two might be better stripped back.  Whatever ... will pick this up on the sweep.